books/RayTracingInOneWeekend.html

```diff
@@ -502,54 +502,93 @@
 
 Sending Rays Into the Scene
 ----------------------------
-Now we are ready to turn the corner and make a ray tracer. At its core, a ray tracer sends rays
-through pixels and computes the color seen in the direction of those rays. The involved steps are
+Now we are ready to turn the corner and make a ray tracer.
+At its core, a ray tracer sends rays through pixels and computes the color seen in the direction of
+  those rays.
+The involved steps are
 
-    1. Calculate the ray from the eye to the pixel,
+    1. Calculate the ray from the “eye” through the pixel,
     2. Determine which objects the ray intersects, and
     3. Compute a color for that intersection point.
 
 When first developing a ray tracer, I always do a simple camera for getting the code up and running.
 I also make a simple `ray_color(ray)` function that returns the color of the background (a simple
-gradient).
+  gradient).
 
 I’ve often gotten into trouble using square images for debugging because I transpose $x$ and $y$ too
-often, so I’ll use a non-square image.
+  often, so I’ll use a non-square image.
 For now we'll use a 16:9 aspect ratio, since that's so common.
+A 16:9 aspect ratio means that the image width over image height = 16/9.
```

Comment:
"A square image has a 1:1 aspect ratio, because it's width is the same as its height. Since we don't want to use a square image, we'll need to choose a different aspect ratio. We'll choose 16:9 because it's so common. A 16:9 aspect ratio means that the ratio of image width to image height is 16:9. As another example, an image 800 pixels wide by 400 pixels high has a 2:1 aspect ratio"


Consider using latex shenanigans for 16:9 and other aspect ratios

Reply:
? Do you mean "use the Unicode ratio glyph"? ( `:` vs `∶`)?

Reply:
That or $16:9$